backend/src/main/kotlin/app/ehrenamtskarte/backend/verification/webservice/schema/CardMutationService.kt

```diff
@@ -23,10 +27,15 @@ class CardMutationService {
             if (!Authorizer.mayCreateCardInRegion(user, targetedRegionId)) {
```

Comment:
We should check at the beginning of this function, whether the CardGenerationModel is valid (ie. if dynamic, then activation secret is present, if static, then it's absent).
For invalid input, throw an exception.

Reply:
error should be thrown either way because of database constraint. I'll still add the check as it's probably better practice to not rely on constraints.